code/UMLEditor.py

```diff
@@ -57,7 +76,9 @@ def print_help_message(command = ""):
 
 # **Write Documentation Here**
 def execute(model:UMLModel, command:str, arguments:list = []):
-    print ("To be implemented")
+    if command == 'exit':
+        prompt_exit(model)
+    # print ("To be implemented")
 
```

Comment:
Oh I forgot to change that back to print ("To be implemented")

Reply:
You can always add another commit to this branch (or someone else can) to revert the behavior